docs/api-reference/static-analysis.md

```diff
@@ -83,66 +84,134 @@ Finally, to make `eslint` easy to run, it is suggested to place commands in `pac
 
-[`eslint-plugin-security`](https://github.com/eslint-community/eslint-plugin-security) is recommended by DBOS and installed with `@dbos-inc/eslint-plugin`.  In particular, the following rules are suggested:
+---
+
+### Recommended rules/plugins:
+
+This includes the base rules and plugins.
+
+These rules are added:
+- `eqeqeq`
+- `@typescript-eslint/no-unused-vars` (silence the rule with leading underscores)
+- `@typescript-eslint/no-for-in-array`
+- `@typescript-eslint/no-misused-promises`
+- `@typescript-eslint/no-floating-promises`
 
-#### `security/detect-unsafe-regex`
-This rule detects regular expressions that may take a long time to run, slowing the event loop.  It is enabled in all DBOS `eslint` configurations.
+These rules are disabled:
+- `semi`
+- `no-empty`
+- `no-constant-condition`
+- `@typescript-eslint/no-unnecessary-type-assertion`
 
-For example, a regular expression that checks password complexity may be used as part of a registration process (and therefore available to the world at large).  If this regular expression involves significant "backtracking", it could take a long time to run on certain user inputs thereby forming a building block of a "denial of service" attack on the system.
+These plugins are added:
+- `eslint:recommended`
+- `@typescript-eslint/recommended`
+- `@typescript-eslint/recommended-requiring-type-checking`
 
-Consideration should also be given to the `detect-non-literal-regexp` rule.
+---
 
-### Suggested `no-secrets` Plugin Rules
+### Extended rules/plugins
 
-[`eslint-plugin-no-secrets`](https://github.com/nickdeis/eslint-plugin-no-secrets) is recommended by DBOS and installed with `@dbos-inc/eslint-plugin`.  This plugin provides one rule:
+- This includes the base and recommended rules, along with the rule [@typescript-eslint/no-shadow](https://typescript-eslint.io/rules/no-shadow/).
 
-#### `no-secrets/no-secrets`
-This rule detects strings that may be security credentials or other secrets, perhaps inadvertently left in the code.  Such credentials may be leaked if the source code is exposed (by the version control system, a disgruntled employee, or generic server breach).   Hard-coded credentials are certainly difficult to track, change, and manage, and should be placed in the environment or other suitable storage.
+---
 
-### `@dbos-inc/eslint-plugin` Rules
+### Handmade rules
 
-Some DBOS lint rules are provided in the [`@dbos-inc/eslint-plugin`](https://github.com/dbos-inc/eslint-plugin) package.
+One handmade lint rule is provided in the [`@dbos-inc/eslint-plugin`](https://github.com/dbos-inc/eslint-plugin) package.  This rule is enabled by default in all `@dbos-inc/eslint-plugin` configurations.
 
 #### `@dbos-inc/detect-nondeterministic-calls`
```

Comment:
Split this into two rules if possible, and if not, call it `dbos-static-analysis`

Reply:
Resolved